java/ci/Dockerfile.centos7

```diff
@@ -36,3 +36,6 @@ RUN cd /rapids/ && wget https://dl.bintray.com/boostorg/release/1.72.0/source/bo
    tar zxf boost_1_72_0.tar.gz && \
    cd boost_1_72_0 && \
    scl enable devtoolset-7 "./bootstrap.sh --prefix=/usr && ./b2 install --with-filesystem threading=multi link=static cxxflags=-fPIC; exit 0"
+
+RUN cd /usr/local/ && wget --quiet https://github.com/Kitware/CMake/releases/download/v3.19.0/cmake-3.19.0-Linux-x86_64.tar.gz && \
+   tar zxf cmake-3.19.0-Linux-x86_64.tar.gz
```

Comment:
The install of cmake3 (and link to /usr/bin) should be removed.

Reply:
Good catch. Removed.